package.json

```diff
@@ -2,27 +2,32 @@
   "name": "so-fetch-js",
   "version": "0.1.0",
   "description": "A nice wrapper around the fetch API with some extras",
-  "main": "lib/index.js",
+  "browser": "umd/so-fetch.umd.js",
```

Comment:
seems like `.umd` is redundant since this file is in `umd/` folder. I'll go with `umd/so-fetch.min.js`